index.d/minishift.yaml

```diff
@@ -11,3 +11,17 @@ Projects:
     build-context: ./
     depends-on:
       - centos/centos:7
+
+  - id: 2
+    app-id: minishift
+    job-id: minishift-build
```

Comment:
@gbraad Can we caller it `minishift-builder` image. IMO it sounds better that way.

Reply:
it is not really a 'builder'... as it is merely supports the process.

Reply:
@gbraad Right, it is going help build of Minishift using this image as it contains golang run time. So I still think `builder` is a better name than `build`.